articles/users/guides/bulk-user-imports.md

```diff
@@ -81,8 +207,32 @@ Furthermore, all of your job-related data is automatically deleted after 24 hour
 
 ## Retrieve failed entries
 
-You can query and retrieve details on failed entries via the API using the [Get Failed Job Error Details endpoint](/api/management/v2#!/Jobs/get_errors).
+If there were errors in the user import job, you can get the error details by making a `GET` request to the [get job error details](/api/management/v2#!/Jobs/get_errors) endpoint. Be sure to replace the `MGMT_API_ACCESS_TOKEN` and `JOB_ID` placeholder values with your Management API Access Token and user import job ID.
+
+```har
+{ 
+  "method": "GET",
+  "url": "https://${account.namespace}/api/v2/jobs/JOB_ID/errors",
+  "headers": [ 
+    { 
+      "name": "Content-Type",
+      "value": "application/json"
+    },
+    { 
+      "name": "Authorization",
+      "value": "Bearer MGMT_API_ACCESS_TOKEN"
+    }
+  ]
+}
+```
 
+If the request is successful, you'll receive a response similar to the following:
+
+```json
+{
+  "TODO": "TODO" // example response
```

Comment:
☝️

Reply:
Here's an example:
```json
[
    {
        "user": {
            "email": "test@test.io",
            "user_id": "7af4c65cb0ac6e162f081822422a9dde",
            "custom_password_hash": {
                "algorithm": "ldap",
                "hash": {
                    "value": "*****"
                }
            }
        },
        "errors": [
            {
                "code": "...",
                "message": "...",
                "path": "..."
            }
        ]
    }
]
```

The `custom_password_hash` field will contain the full hash object, but the sensitive fields will be redacted (e.g `hash.value: '*****'`)